packages/cubejs-mysql-driver/src/MySqlDriver.ts

```diff
@@ -112,7 +112,7 @@ export class MySqlDriver extends BaseDriver implements DriverInterface {
         }
         return true;
       },
-      destroy: (connection) => promisify(connection.end.bind(connection))(),
+      destroy: (connection) => promisify(connection.end.bind(connection)),
```

Comment:
Hello!

What issue do you try to resolve? Because it looks strange, because `destroy` should return a Promise, but without calling promisified connection. end, it will return function. for example:

````js
const pool = { destroy: (connection) => connection.end };
pool.destroy({ end: async () => {} })
````

result:

```js
async () => {}
```

Thanks

Reply:
I do some tests with MySQL like databases.  when fetching database tables it will block. and I do some tests find that caused by connection pool destroy method . 

and  in ` testConnection `
https://github.com/cube-js/cube.js/blob/master/packages/cubejs-mysql-driver/src/MySqlDriver.ts#L173
it call like this ` await (<any> this.pool)._factory.destroy(conn);` 
so I think this would  return promise  method with one argument .

Reply:
and  this  also works 
```
const pool = { destroy: (connection) => connection.end() };
````

Reply:
or  destroy would like this 

```code
destroy: (connection) => {promisify(connection.end.bind(connection))()}
```

Reply:
@ovr 

same as JDBC driver code

https://github.com/cube-js/cube.js/blob/master/packages/cubejs-jdbc-driver/src/JDBCDriver.js#L81

```code
      destroy: async (connection) => promisify(connection.close.bind(connection)),
```
I saw the current connect pool destroy method not all the same